### DISCUSSION TITLE
Build Difficulties / Differences with PROTEAR Headphones

### DISCUSSION THREAD

**Discussion by ds-trt** [sparkfun/SuperHeadphones#2] [Q&A - Build Questions/Issues]
I’m trying to assemble a pair of your headphones, but the smoostart headphones were not available. I picked up a pair of the second recommended PROTEAR headphones on Amazon. Instead of the 3 AAA batteries of the smoostart muffs the PROTEAR have 2 AA batteries. Will the nominal 3V work for the 3.3v chips?  The ESP32 data sheet shows 3.0-3.6 volt range but I don’t see any voltage regulator to handle what I presume would be 4.5 volts from the 3 AAA batteries in your design, but also concerned that the 2AA batteries would drop below 3 volts as batteries wear and result in brown out in the chips? Any thoughts appreciated.

Also, FYI, the angle between the potentiometer and the audio in jack is different on the PROTEAR so your PCB does not work for both. Not a big deal for me as I don’t need the audio in, but just thought I’d let you know.

**Comment by lewispg228** [comment]
Hi Shelley,
Thanks for reaching out. Happy to help!

First of all, Sorry we didn't mention that the custom PCB is specifically for the [Smoostart](https://www.amazon.com/Smoostart-Cancelling-Protection-Snowblowing-Construction/dp/B08M95FP4X) design, and using anything else (like the [PROTEAR](https://www.amazon.com/dp/B09ZPBM9F2/?th=1)s) will most likely need a tweak. We will add in a note to the tutorial for future readers.

About your power. Your brown-out concerns are valid. I would not recommend powering from a 2xAA pack.  As the pack drains just a little bit below fully charged, then you will run into issues with things browning-out. Also, on the "mother board" PCB design, that battery pack voltage connects to the VBATT input on the ESP32 Thing Plus C. This eventually hits a 3.3V voltage regulator, that needs to see something above 3.3V to function properly.

Are you using the [ESP32 Thing Plus C](https://www.sparkfun.com/products/20168)? If so, I would recommend switching to a lipo battery. The [400mAh capacity battery](https://www.sparkfun.com/products/13851) we sell should fit okay in the current compartment of the PROTEARs. I am currently using this battery on my latest prototype. I switched to a lipo battery, so I could use the on-board charger on the dev board (in my case it is the [ESP32 Qwiic Pocket](https://www.sparkfun.com/products/22925), but the ESP32 Thing Plus C also has the charger IC).

A single cell Lipo will give you 3.7-4.2 volts, and then the on-board regulator will get that down to a solid 3.3V (no matter how charged your battery is).

If you switch to a lipo, then you no longer need the isolation circuit that is on the custom "mother board".


![image](https://github.com/sparkfun/SuperHeadphones/assets/601236/fabe04b0-0ffb-4a2d-aacc-3efdf5c3f0b8)


Did you buy a copy of the custom PCB yet? If not, then maybe we can work together to tweak it for the lipo and the PROTEARs. 

If you do already have a copy of the PCB, then we can also work towards a working circuit by cutting a couple traces and hand soldering a couple connections (essentially, just removing the isolation circuit (Q1) and wiring the battery to the switch-terminal on the potentiometer) - this is what I did on my latest set. 

Let us know and we look forward to hearing from you.

Cheers,
Pete

**Reply by ds-trt** [reply]
I am using the Thing Plus C, and I very much like the idea of going to lipo. I have a 1000mAh cell on hand that I'll try to use. Did you mount yours in the battery side or fit it in with everything else on the motherboard side?

I do already have the PCB's so I am interested in what needs to happen to remove the isolation circuit. I am also interested in tweaking the PCB for the PROTEAR's. But, for now it makes sense to continue with what I have.

FYI, I found what looks to be a rebranded version of the Smoostarts, they do have 3 aaa batteries, and if the angle is the same between the pot and the line in, that could be a good option for people with the current PCB. I'll let you know when they arrive.

As always, I appreciate the help.

**Reply by lewispg228** [reply]
Hi @ds-trt ,
Sorry for the delay in my response here. Hope you are doing well and happy new year!

I took another look at the mother-board design, and you don't have to cut any traces to wire up a lipo battery. You simply need to omit populating R1 and Q1. Then you can wire your lipo battery up to the potentiometer/switch pins. Hopefully this drawing will help:

![image](https://github.com/sparkfun/SuperHeadphones/assets/601236/4404c778-59c3-4f34-81a2-cbb7a1bc3d3e)

Note, with this setup, in order to charge the battery, the potentiometer switch will need to be clicked "on".

Optionally, you can just plug your lipo battery into the Thing Plus battery connector, but this will always be "ON", and so if you'd like to shut your headphones completely "off", then you'd have to disconnect the battery manually to do this.

Also, just to double check, you do have PCB "v20", correct? (There should be a small "v20" in the bottom copper)

In other news, I received a pair of the PROTEARS. I initially gave them a listen, and they do not have much bass performance. The drivers are small and do not have any opening in the mounting housing. This is most likely the cause of the poor low frequency response. I may experiment with creating larger mounts that include bass ports, and also just replacing the drivers with larger. That aside, they are quite comfortable and look to be pretty good build quality. The space for the electronics is lower profile than the smoostarts, and at this point I'm not entirely sure you/we can fit our replacement PCB+thing-plus+codec in there. 

In yet other news, I've been troubleshooting some RF intereference. I've reached out to a [DIY mic builders group](https://groups.io/g/MicBuilders/topic/suppressing_rf_noise/103902763), and hope to fine a solution. Currently, it seems to be isolated to my drum rehearsal spot, because in most other places (my basement, house, work), I am not experiencing RF interference. I'll update here once I/we find the problem and a working solution.

Hope your build is going okay, and let me know how the lipo battery works out.

Cheers,
Pete
